ledger/trie.go

```diff
@@ -187,13 +187,13 @@ type TrieProof struct {
 }
 
 // NewTrieProof creates a new instance of Trie Proof
-func NewTrieProof() *TrieProof {
+func NewTrieProof(pathByteSize int) *TrieProof {
```

Comment:
shouldn't we read this from the common and not pass it ?

Reply:
I made that refactor mainly to combine [these lines](https://github.com/onflow/flow-go/blob/master/ledger/complete/mtrie/forest.go#L260-265) together:  create the proofs directly with the right length, instead of creating them with length `0` and then update their length. 

In https://github.com/onflow/flow-go/pull/514, that proof length is gonna become be a constant and we don't need to pass it anymore.